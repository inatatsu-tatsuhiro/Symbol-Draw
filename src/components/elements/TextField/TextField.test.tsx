import React from 'react'
import { render } from '@testing-library/react'

import Component from './index'

const text = 'TEXT FIELD TEXT'

test('text field: matching text', () => {
  const { container } = render(<Component text={text} inputRef={null} />)
  expect(container.innerHTML).toMatch(text)
})

test('text field: snapshot testing', () => {
  const { asFragment } = render(<Component text={text} inputRef={null} />)
  expect(asFragment()).toMatchSnapshot()
})
