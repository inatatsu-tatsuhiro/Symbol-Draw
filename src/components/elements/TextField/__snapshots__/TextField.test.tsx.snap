// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text field: snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="MuiFormControl-root MuiFormControl-fullWidth css-q8hpuo-MuiFormControl-root"
  >
    <label
      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
      data-shrink="false"
      for="outlined-adornment-password"
    >
      TEXT FIELD TEXT
    </label>
    <div
      class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
    >
      <input
        aria-invalid="false"
        class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
        type="text"
        value=""
      />
      <fieldset
        aria-hidden="true"
        class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="css-173wfxe"
        >
          <span>
            TEXT FIELD TEXT
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</DocumentFragment>
`;
